.travis.yml

```diff
@@ -9,8 +9,8 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-        - php: 7.0
+        - php: '7.0'
```

Comment:
Ah you're right, they changed it in their doc as well. I don't think it's luck though, they did document it without quotes before.